src/main/java/com/minecolonies/coremod/entity/ai/citizen/alchemist/EntityAIWorkAlchemist.java

```diff
@@ -510,7 +512,7 @@ private IAIState checkBrewingStandFuel()
 
         if(!InventoryUtils.hasItemInItemHandler(worker.getInventoryCitizen(), Items.BLAZE_POWDER)
              && !InventoryUtils.hasItemInProvider(building, Items.BLAZE_POWDER)
-             && !building.hasWorkerOpenRequestsOfType(worker.getCitizenData().getId(), TypeToken.of(StackList.class)))
+             && !building.hasWorkerOpenRequestsOfType(worker.getCitizenData().getId(), TypeToken.of(Stack.class)))
```

Comment:
That's def the reason, copy pasta error. But then we need to use the code other workers use too where we check if the request contains the item

Reply:
I looked at other places `hasWorkerOpenRequestOfType` was called and they don't check.

Reply:
And it doesn't look like the AI creates any other requests at all, let alone other `Stack` requests, so it may be safe.  Provided it doesn't collide with whatever the crafting system does.  (But at worst that should presumably only cause the requests to be processed sequentially rather than in parallel.)